app/serializers/move_serializer.rb

```diff
@@ -10,13 +10,19 @@ class MoveSerializer < ActiveModel::Serializer
   has_one :prison_transfer_reason, serializer: PrisonTransferReasonSerializer, if: -> { object.prison_transfer_reason.present? }
   has_many :documents, serializer: DocumentSerializer
   has_many :court_hearings, serializer: CourtHearingSerializer
+  belongs_to :allocation, serializer: AllocationSerializer
 
   INCLUDED_ATTRIBUTES = {
-    person: %i[first_names last_name date_of_birth assessment_answers indentifiers ethnicity gender reason_comment],
-    from_location: %i[location_type description],
-    to_location: %i[location_type description],
-    documents: %i[url filename filesize content_type],
-    prison_transfer_reason: %i[key title],
+    person: %i[ethnicity gender],
```

Comment:
I'm a bit worried that these serialization changes will have an impact on the front end and also on other api consumers... are they intended and have they been discussed?

Reply:
Discussed this offline with @cwrw - essentially the output of the endpoint hasn't changed (other than to include the allocation where it's related to a move). The previous implementation of included attributes did not work as suggested by the code - instead every relationship included all of the attributes. That was easy to see from testing with Postman.

I prefer that the code here now matches the actual behaviour. It's something we can revise further moving forwards, perhaps when we move to an alternate JSONAPI technique and allow the API client to specify attributes it wishes to receive.

Reply:
Also `ethnicity` and `gender` are special cases - those are related models rather than attributes, so need to be explicitly included, otherwise the associated models don't get included in the response. Think of this as deep nesting of models 🤔